### DISCUSSION TITLE
我的耕种奖励目标地址已经收到0.25xch，但是钱包没有收到奖励（钱包地址已同步）

### DISCUSSION THREAD

**Discussion by flynesses** [Chia-Network/chia-blockchain#8207] [Support]
### Discussed in https://github.com/Chia-Network/chia-blockchain/discussions/8187

<div type='discussions-op-text'>

<sup>Originally posted by **flynesses** August 24, 2021</sup>
我的耕种奖励目标地址已经收到0.25xch，但是钱包没有收到奖励（钱包地址已同步）
![image](https://user-images.githubusercontent.com/39182553/130557067-506d2dc8-b8ac-42df-93b6-809dcca08712.png)
![image](https://user-images.githubusercontent.com/39182553/130557111-32e27c24-347c-496b-a630-7b03d246cca8.png)
</div>

**Comment by loppefaaret** [comment]
This would indicate that a plot tied to a plotnft has won - if the plotnft was pointed to a pool, the pool would have the rights to the 1.75 part of the reward - if the plotnft is self-farming, you might need to go to the "Pool" tab, and click "Claim Rewards" on the plotnft to claim your 1.75 pool reward!

**Comment by emlowe** [comment]
Closing issue - please use support discussion

**Comment by BrandtH22** [comment]
@flynesses , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
